dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -80,13 +83,21 @@
         this.maxQueueSize = maxQueueSize;
         this.queueSizeCounter = metrics.requestsQueued(channelName);
         this.queuedTime = metrics.requestQueuedTime(channelName);
+        this.limitedResultSupplier = () -> Futures.immediateFailedFuture(new SafeRuntimeException(
+                "Unable to make a request",
+                SafeArg.of("queueSizeEstimate", queueSizeEstimate.get()),
```

Comment:
I think we should remove this value, reading from the estimate will race execution and cause confusion.